axis_backend/apps/clients/serializers/industry_serializer.py

```
@@ -0,0 +1,241 @@
+
+    class Meta:
+        model = Industry
+        fields = [
+            'id',
+            'name',
+            'code',
+            'parent_name',
+            'client_count',
+            'created_at',
+            'updated_at',
+        ]
```

Ruff (0.14.5)

Mutable class attributes should be annotated with `typing.ClassVar`

(RUF012)